handlers/meshery_application_handler.go

```diff
@@ -18,6 +18,20 @@ type MesheryApplicationRequestBody struct {
 	ApplicationData *models.MesheryApplication `json:"application_data,omitempty"`
 }
 
+// swagger:route POST /api/experimental/application/deploy ApplicationAPI idPostApplicationFile
+// Handle POST request for Application File Deploy
+//
+// Deploy an attached application file with the request
+// response:
+// 	200:
```

Comment:
I am damn sure there are parameters(in body) that this handler must be expecting, because we usually attach the file in form of body with the request.

Reply:
<img width="954" alt="Screenshot 2021-07-25 at 5 58 04 PM" src="https://user-images.githubusercontent.com/64970095/126898934-19fd8f8e-e19d-424d-8913-03ac590122d7.png">
Hey @piyushsingariya does this look good to you ?

Reply:
Not sure, but could you match it to Pattern file POST method?

Reply:
Also, add the Tag as ApplicationsAPI, the plural would be better.

Reply:
Oh yes I will do it.

Reply:
Pattern file POST method you mean this endpoint `api/pattern/deploy`?

Reply:
<img width="1410" alt="Screenshot 2021-07-25 at 8 32 21 PM" src="https://user-images.githubusercontent.com/64970095/126903741-051bfa57-cbfb-452a-a548-a0f9c62fc48f.png">

Reply:
This file is not accepting anything.🤔

Reply:
We gotta fix that in both places. Can you go through the handler for pattern deploy, We do pass a structured JSON on the deploy URL.

Reply:
So like I need to change the `api/pattern/deploy` in the same way I did for `api/experimental/application/deploy`?

Reply:
You don't need to change, you need to figure out what exactly are the parameters consumed by these endpoints.

Reply:
Hey @piyushsingariya I checked the pattern API and found that the JSON response in the Applications API is same but the PatternAPI is not taking any parameter in it so that the only difference I see over here. So I think the parameter should be removed I guess because as fas as I see the`api/experimental/application/deploy` is not expecting any parameters to be passed upon.

Reply:
Ok